### PR TITLE
Collapsing nav fixed

### LINKED ISSUE #98: Navigation Bar Don't Collapse in Mobile Biew
**Describe the bug** :bug:
When you navigate to another webpage, then navigation Bar don't collapse in Mobile View.

**To Reproduce** :mag:
1. Go to 'mobile view'
2. Click on 'learn'
3. Scroll down to 'bottom'
6. See an error

**Expected behavior** :honeybee:
The Navigation bar has to collapse after clicking on the nav link.

Screenshot_2021-04-10-21-44-00-898_com android chrome

**Smartphone (please complete the following information):** :iphone:

### PR DESCRIPTION
**Pull request by kumar-tushar**
Closes #98

BEFORE
https://drive.google.com/file/d/17KhXeY-94M9Z3bndy-pD-o8dqNGXMJdn/view?usp=sharing

### CODE REVIEW

**Review by apurv-1 — COMMENTED** [review]
Both before and after links are same.

**Comment by kumar-tushar** [comment]
AFTER
https://drive.google.com/file/d/17JavIe-jiu89oqPDFJI65WXEc9HA1wXm/view?usp=drivesdk

**Review by apurv-1 — APPROVED** [review]

File: client/src/components/navbar/navbar.js [hunk 1]
```diff
@@ -17,52 +17,32 @@ const renderTooltip = props => {
...
+            <OverlayTrigger placement = 'right-end' overlay = { renderTooltip }>
+              <img src = { pbchessLogo } width = '60em' height = '60em' className = 'd-inline-block align-top' alt = 'pbchess logo'/>
+            </OverlayTrigger>
+          </Link>
+
+        </Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav"/>
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="ml-auto">
+            <Nav.Item><Nav.Link href="/play">Play</Nav.Link></Nav.Item>
```
Comment (fave77) [inline client/src/components/navbar/navbar.js]: @kumar-tushar By not using `Link` from `react-router-dom`, it defeats the purpose of SPA and faster loads. As you can see Nav.Link's href introduces a delay in page load!

**Review by fave77 — CHANGES_REQUESTED** [review]
@kumar-tushar please try to come with other alternatives for collapsing the navbar

File: client/src/components/navbar/navbar.js (hunk 1 above)
Comment (kumar-tushar) [reply client/src/components/navbar/navbar.js]: @fave77 Hello Sir, I have made some changes, Please have a look.

Comment (kumar-tushar) [reply client/src/components/navbar/navbar.js]: https://drive.google.com/file/d/18EV7j18GiFHd6R9bpXFjwGwxZBsW7tj1/view?usp=sharing

File: client/src/components/navbar/navbar.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { Link,  } from 'react-router-dom';
```
Comment (fave77) [inline client/src/components/navbar/navbar.js]: remove `,`

File: client/src/components/navbar/navbar.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 import Nav from 'react-bootstrap/Nav';
 import NavDropdown from 'react-bootstrap/NavDropdown';
 import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
 import Tooltip from 'react-bootstrap/Tooltip';
-
 import pbchessLogo from '../../images/pbchess-logo.svg';
-
 import './navbar.css';
+import { LinkContainer } from "react-router-bootstrap";
+import Home from "../home/home";
```
Comment (fave77) [inline client/src/components/navbar/navbar.js]: why are you importing `home` component?

File: client/src/components/navbar/navbar.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 import Navbar from 'react-bootstrap/Navbar';
 import Nav from 'react-bootstrap/Nav';
 import NavDropdown from 'react-bootstrap/NavDropdown';
 import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
 import Tooltip from 'react-bootstrap/Tooltip';
-
 import pbchessLogo from '../../images/pbchess-logo.svg';
-
 import './navbar.css';
+import { LinkContainer } from "react-router-bootstrap";
```
Comment (fave77) [inline client/src/components/navbar/navbar.js]: - please import this on the 3rd line
- use the single quote for consistency 
- only update `package.json` and `package-lock.json` while using a new package

**Review by fave77 — CHANGES_REQUESTED** [review]
@kumar-tushar Please do not change the existing structure of the code present in the file and only add your own changes. 

Lastly, before making a PR please run `npm test` to see if you introduced any breaking changes!

**Review by kumar-tushar — COMMENTED** [review]
done

**Pull request closed without merging**